website/static/js/share/stats.js

```diff
@@ -121,6 +124,7 @@ Stats.controller = function(vm) {
             url: '/api/v1/share/stats/?' + $.param({q: self.vm.query()}),
             background: true
         }).then(function(data) {
+            data.charts.shareDonutGraph.onclick = function (d, element) { return getSourceName(d, element, self.vm) };
```

Comment:
It would be more appropriate to add the onclick function [here](https://github.com/erinspace/osf.io/blob/feature/share/search_updates/website/static/js/share/stats.js#L23)